storm.go

```diff
@@ -152,12 +153,26 @@ func toBytes(key interface{}, codec codec.MarshalUnmarshaler) ([]byte, error) {
 	if key == nil {
 		return nil, nil
 	}
-	if k, ok := key.([]byte); ok {
-		return k, nil
-	}
-	if k, ok := key.(string); ok {
-		return []byte(k), nil
+	switch t := key.(type) {
+	case []byte:
+		return t, nil
+	case string:
+		return []byte(t), nil
+	case int:
```

Comment:
I thought the same at first, but the problem is that if i collapse, the `t` var won't be cast and i would have to reinfer it and cast it to int: https://play.golang.org/p/gMaR3HdZvr
and i can't use `fallthrough` either.
I'm gonna add the missing integer types, it's gonna be a long function...